pkg/network/infraconfigurators/macvtap.go

```diff
@@ -73,9 +64,9 @@ func (b *MacvtapPodNetworkConfigurator) DiscoverPodNetworkInterface(podIfaceName
 	return nil
 }
 
-func (b *MacvtapPodNetworkConfigurator) podIfaceMAC() string {
-	if b.mac != nil {
-		return b.mac.String()
+func (b *MacvtapPodNetworkConfigurator) retrieveVmMac() string {
```

Comment:
About getters in golang: https://golang.org/doc/effective_go#Getters

I've tried to follow that when I named that method - "retrieve" sounds like a fancy "get" . 

I'm OK with renaming from `podIfaceMAC` to `vmMac` .

Reply:
Removed the change from this commit. Added a fix regrading macvtap mac to `NetworkConfigurator: calculate mac address closer to the usage ` commit.